cassandra-mesos-scheduler/src/main/java/io/mesosphere/mesos/frameworks/cassandra/scheduler/PersistedCassandraFrameworkConfiguration.java

```diff
@@ -26,7 +26,12 @@
 import org.jetbrains.annotations.NotNull;
 import org.joda.time.Duration;
 
+import java.util.List;
+
+import static io.mesosphere.mesos.frameworks.cassandra.CassandraFrameworkProtos.ExternalDc;
+
 public final class PersistedCassandraFrameworkConfiguration extends StatePersistedObject<CassandraFrameworkConfiguration> {
+    @NotNull private final SeedManager seedManager;
```

Comment:
This class should not have a service dependency other than `State`.

This field should be passed to `CassandraCluster` as a constructor argument that can then be set to a field.